docs/4. API/Server/Session-based verification/6-create2-verification.mdx

```diff
@@ -0,0 +1,96 @@
+---
+id: session-create2-verification
+title: Create2 Verification
+slug: /api/server/session/verify/create2
+---
+
+import V2Note from "./_v2Note.mdx";
+
+# Create2 Verification
+
+Verify counterfactual smart contracts created with the opcode `CREATE2` with session.
+
+<V2Note />
+
+**URL** : `/session/verify/create2`
+
+**Method** : `POST`
+
```

Comment:
Can we add an example request here? It's not really clear how the body should be formatted such as the `constructorArgs`.

Reply:
done